docs/en/operation/User-CLI.md

```diff
@@ -1291,7 +1291,7 @@ but the actual data may be deleted a while later.
 
 * Adding `-R` option deletes all contents of the directory and the directory itself.
 * Adding `-U` option skips the check for whether the UFS contents being deleted are in-sync with Alluxio
```

Comment:
sounds like we should make this the default option when we have a backcompat breaking release (ex. 3.0). i heard that @apc999  has a list of backcompat debt somewhere

Reply:
We are going to totally rewrite this part of logic (but this is not immediately happening).